tests/array.rs

```diff
@@ -1372,6 +1424,19 @@ fn to_owned_neg_stride() {
     assert_eq!(c, co);
 }
 
+#[test]
+fn discontiguous_owned_to_owned() {
+    let mut c = arr2(&[[1, 2, 3],
+                       [4, 5, 6]]);
+    c.slice_inplace(s![.., ..;2]);
+
+    let co = c.to_owned();
+    assert_eq!(c.strides(), &[3, 2]);
+    assert_eq!(co.strides(), &[2, 1]);
+    assert_eq!(c.shape(), co.shape());
+    itertools::assert_equal(c.iter(), co.iter());
```

Comment:
I'd prefer just `assert_eq!(c, co)` instead of `itertools::assert_equal()`, unless there's a specific reason for using `itertools::assert_equal()` here.

Reply:
No reason, I think I was just confused and thought PartialEq would check strides for some reason